helm/pinot/templates/_helpers.tpl

```diff
@@ -319,3 +319,14 @@ The name of the pinot minion stateless config.
 {{- define "pinot.minionStateless.config" -}}
 {{- printf "%s-config" (include "pinot.minionStateless.fullname" .) | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
+
+{{/*
+Return pinot namespace to use
+*/}}
+{{- define "pinot.namespace" -}}
+{{- if .Values.namespaceOverride -}}
+    {{- .Values.namespaceOverride -}}
+{{- else -}}
+    {{- .Release.Namespace -}}
```

Comment:
Is this k8s system default namespace?
what's the behavior if user set the default namespace to something else?

Reply:
Here is the behaviour
- If the `namespaceOverride` value is provided, it will precede the rest.
- If the `namespaceOverride` value is not provided, and we use the helm command with `-n <value>`, the provided value in the command will get used.
- If both the above values are not provided, then the `default` value will be set as a namespace.